runtime/compiler/control/CompilationThread.cpp

```diff
@@ -2846,6 +2846,8 @@ void TR::CompilationInfo::stopCompilationThreads()
       fprintf(stderr, "numVirtualMethodsValidationSucceeded: %d\n", aotStats->virtualMethods.numSucceededValidations);
       fprintf(stderr, "numInterfaceMethodsValidationFailed: %d\n", aotStats->interfaceMethods.numFailedValidations);
       fprintf(stderr, "numInterfaceMethodsValidationSucceeded: %d\n", aotStats->interfaceMethods.numSucceededValidations);
+      fprintf(stderr, "numAbstractMethodsValidationFailed: %d\n", aotStats->abstractMethods.numFailedValidations);
```

Comment:
Initially this was part of a separate commit to enable abstract method inlining. However, if I enabled it in the old infrastructure, it started crashing. Hence, I grouped this change along with the change to add the infra to allow abstract method inlining.

Reply:
Heh, if it makes you feel any better, abstract method inlining definitely works with the new infrastructure, because it was the source a significant amount of performance gain.